src/public/meeting.ts

```diff
@@ -110,6 +110,13 @@ export namespace meeting {
      */
     isAppSharing: boolean;
   }
+  export interface ISpeakingState {
+    /**
+     * indicates whether one or more participants in a meeting are speaking, or
```

Comment:
Hi @ydogandjiev, I included the comment for ISpeakingState here on the following lines.